documentation/Using-the-SDK/Creating-a-simple-sdk-plugin.md

```diff
@@ -1,75 +1,98 @@
-# Creating a Simple Custom Data Source
+# Creating a Simple SDK Plugin
 
-A Custom Data Source (CDS) uses the SDK to create a data processing pipeline for any data sources, typically files, 
-that the CDS supports. Notably, a CDS acts as the entry point for the SDK runtime to discover and utilizes these data 
-pipelines. An SDK Plugin may contain more than one CDS.
+An SDK uses the SDK to create a mapping between arbitrary data sources (e.g. files with 
+specific formats) and 
+* zero or more __tables__
+* zero or more __data outputs__
 
-A standard CDS will utilize a Custom Data Processor (CDP) to process the data sources that the SDK runtime provides it.
 
-A plugin may also define tables that can be used by viewers such as Windows Performance Analyzer (WPA) to display the 
+A `CustomDatasource` (CDS) acts as the entry point for the SDK runtime to discover and utilizes these mappings. 
```

Comment:
instead of saying utilizes the mappings, maybe say the data is mapped?

Reply:
The subject of this sentence got swapped at the end. Changing "utilizes" to "create" fixes it I believe.